app/backend/submissions/views.py

```diff
@@ -186,7 +186,7 @@ class SubmissionListAPIView(ListAPIView):
     get: returns a list of well activity submissions
     """
 
-    permission_classes = (WellsEditPermissions,)
+    permission_classes = (WellsSubmissionPermissions,)
```

Comment:
We should check with Lindsay but I think WellsEditPermissions is more appropriate for List and Get. I think we only want people with the "well submission" role to be able to submit reports, not list other reports.

Not sure if it's better to change the views to support different permission classes in a view class (for making new submissions vs listing them) or to change the permission class to only allow POST requests unless the user has edit rights.

Reply:
I added a new role wells_submission_viewer which is required to access the get and list endpoints.